### PR TITLE
CB-14796 Remove RBAC entitlement and remove legacy authorization code

### PR DESCRIPTION
**Pull request by horadla23**
See detailed description in the commit message.

### CODE REVIEW

**Review by bergerdenes — APPROVED** [review]
Minor nits in readme formatting

File: authorization-common/README.md [hunk 1]
```diff
@@ -254,13 +252,12 @@ You can support authorization on new resources, and you can specify the rights a
 
 1. Define new resource types in `AuthorizationResourceType`,
 2. define new rights by extending `AuthorizationResourceAction` enum with new (String right, AuthorizationResourceType type) values (the new right should be defined in UMS previously),
-3. to support legacy authorization add a mapping from the new right to the legacy right in the `legacyRights.json`,
-4. implement `ResourceBasedCrnProvider`'s methods to support different scenarios on API level,
+3implement `ResourceBasedCrnProvider`'s methods to support different scenarios on API level,
```
[the remark this answers is not in the document]
Comment (horadla23) [reply authorization-common/README.md]: good catch, thx!

**Review by topolyai5 — APPROVED** [review]

**Review by lnardai — APPROVED** [review]

**Comment by lnardai** [comment]
Please disable legacy real-ums-test before merge.

**Comment by horadla23** [comment]
> Please disable legacy real-ums-test before merge.

I have removed references for legacy authz test suite in jenkins jobs and removed the related suite file and test in this PR.

**Pull request merged**